infection.json.dist

```diff
@@ -2,6 +2,13 @@
     "source": {
         "directories": [
             "src"
+        ],
+        "excludes": [
+            "Controller",
+            "DependencyInjection/SetonoSyliusOrderEditExtension.php",
+            "Exception",
+            "EventSubscriber",
+            "Form"
```

Comment:
Instead of this I will just lower the msi's. It's not a problem doing that. This something we can review by the end of project, e.g. looking over mutants to see if there are some important ones

Reply:
On the other hand, these are directories that would not be directly covered with tests. Unit testing forms, event subscribers, or exceptions is redundant, as they're well covered by functional tests, so from my perspective it's better to explicitly show we don't want them to be mutated. I'm open for the discussion, obviously 🖖

Reply:
Now that the functional tests are written with PHPUnit and Infection runs PHPUnit by default, my take is that we can just as well include everything in infection and just lower the MSI accordingly. This will provide insights later on as to where our tests are lacking. By excluding, you won't get these insights from Infection.

Reply:
Ok, let's do it your way, I will lower the MSI and let's see how it goes 🫡